tests/workflows/basic_subworkflow_deployment/tests/test_workflow.py

```diff
@@ -495,3 +497,86 @@ def mock_resolve(deployment_name: str, release_tag: str, state: Any):
             workflow_definition=workflow.__class__, span_id=uuid4()
         ).workflow_definition.model_dump()
     )
+
+
+def test_stream_workflow__emits_workflow_initiated_and_rejected_events_on_invalid_inputs(mocker):
+    """
+    Tests that workflow initiated and rejected events are emitted when a subworkflow
+    deployment node has invalid inputs (interface mismatch).
+    """
+
+    # GIVEN a simple workflow that will fail due to invalid inputs
+    class FailingWorkflowInputs(BaseInputs):
+        city: str
+        date: str
```

Comment:
DevinAI we can reproduce the `Missing required input variable` error by changing one of these two input names, for example, from `date` -> `day`